_themes/conan_theme/versions.html

```diff
@@ -13,11 +13,14 @@
         {% endif %}
     {% endfor %}
     <br>
+        <details>
```

Comment:
I think changes to the template should go to master branch, because the template is taken from there if I recall correctly?